cmd/hotfix/msgpack-20180824/main.go

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright 2018 The CovenantSQL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"encoding/binary"
+	"flag"
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/CovenantSQL/CovenantSQL/crypto/kms"
+	"github.com/CovenantSQL/CovenantSQL/kayak"
+	"github.com/CovenantSQL/CovenantSQL/proto"
+	"github.com/CovenantSQL/CovenantSQL/sqlchain/storage"
+	ct "github.com/CovenantSQL/CovenantSQL/sqlchain/types"
+	"github.com/CovenantSQL/CovenantSQL/utils"
+	"github.com/CovenantSQL/CovenantSQL/utils/log"
+	wt "github.com/CovenantSQL/CovenantSQL/worker/types"
+)
+
+var (
+	dhtFile    string
+	privateKey string
+)
+
+func init() {
+	flag.StringVar(&dhtFile, "dhtFile", "dht.db", "dht database file to fix")
+	flag.StringVar(&privateKey, "private", "private.key", "private key to use for signing")
+}
+
+type OldBlock ct.Block
+
+func (b *OldBlock) MarshalBinary() ([]byte, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method OldBlock.MarshalBinary should have comment or be unexported